packages/styles/src/components/card.scss

```diff
@@ -10,26 +10,79 @@
 @use './../mixins/icons' as icons-mx;
 @use './../variables/icons';
 @use './../variables/animation';
+@use '../mixins/utilities' as utilities-mx;
 
 @use './../functions/tokens';
 @use './../tokens/utilities';
+@use './../tokens/components';
 
-tokens.$default-map: utilities.$post-spacing;
+tokens.$default-map: components.$post-cards;
 
 .card-group {
   margin-top: commons.$border-width;
   margin-left: commons.$border-width;
-  box-shadow: commons.$box-shadow-lg;
+  box-shadow: tokens.get('card-interactive-elevation');
```

Comment:
This shadow look a little strange because of a 1px margin on the cards, can you update that please?

Also, if the group contains only non-interactive cards it should probably not have a shadow.